queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -150,6 +169,12 @@ def process(email_info: dict, token: str) -> dict:  # pylint: disable=too-many-l
         region = pycountry.subdivisions.get(code=f'{country_code}-{region_code}')
     foreign_jurisdiction = f'{region.name}, {country.name}' if region else country.name
 
+    # get template and fill in parts
+    template = Path(f'{current_app.config.get("TEMPLATE_PATH")}/CONT-IN-{status}.html').read_text()
+    filled_template = substitute_template_parts(template)
+    jnja_template = Template(filled_template, autoescape=True)
```

Comment:
I simply moved a few things around to make sense, ie:
1. first prepare the variables
2. then get the template
3. finally, render it